modules/oe_whitelabel_helper/src/Plugin/better_exposed_filters/sort/FloatEndSortWidget.php

```diff
@@ -22,10 +22,17 @@ class FloatEndSortWidget extends DefaultWidget {
    */
   public function exposedFormAlter(array &$form, FormStateInterface $form_state) {
     parent::exposedFormAlter($form, $form_state);
-    $form['#attributes']['class'][] = 'float-lg-end';
-    $form['#attributes']['class'][] = 'd-none';
-    $form['#attributes']['class'][] = 'd-md-flex';
-    $form['#attributes']['class'][] = 'align-items-baseline';
+    $form['sort_bef_combine']['#wrapper_attributes']['class'][] = 'float-md-end';
```

Comment:
this is already better than what was done before in this PR and for our purpose ok for now, BUT we should leave a todo to improve this using '#theme' like done in the other plugins (see Drupal\better_exposed_filters\Plugin\better_exposed_filters\sort\) AND we should only do this once we have the listing pages sort themed and merged so we have a unified theming, no duplications (so mention also the sort theming ticket)